apiserver/facades/client/modelconfig/modelconfig.go

```diff
@@ -35,22 +39,23 @@ type ModelConfigAPIV3 struct {
 
 // NewModelConfigAPI creates a new instance of the ModelConfig Facade.
 func NewModelConfigAPI(
+	modelUUID string,
```

Comment:
This should be `coremodel.UUID`. Can we also rename this to modelID. I will be making a change soon to rename the model type from UUID to ID. This just helps make the change easier.